### PR TITLE
python-build: ux updates + bug fixes + recoil-pattern tool

### PR DESCRIPTION
**Pull request by krampus-nuggets**
Hey, the following updates can be found in this pull request:

* Pattern-Tracker tool for generating recoil-patterns.
* Moving banners to external module.
* Update python and main README files.
* Tool state updates output to one-line source.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ This repository contains an Autohotkey script to help you minimize weapon recoil
 
 ### [AHK Script - Detail and Usage 详细使用指南](https://github.com/mgsweet/Apex-NoRecoil-2021/tree/main/AHK)
 
+## Update
```
Comment (mgsweet) [inline README.md:16]: what is this for?

File: README.md [hunk 2]
```diff
@@ -13,6 +13,8 @@ This repository contains an Autohotkey script to help you minimize weapon recoil
...
+=======
```
Comment (mgsweet) [inline README.md:17]: also this one

**Review by mgsweet — CHANGES_REQUESTED** [review]
check README review

**Review by krampus-nuggets — COMMENTED** [review]
I've removed the unneeded heading. Not sure what happened, I blame morning brain.

**Comment by krampus-nuggets** [comment]
The following additional updates have been pushed as well:

* External config for easier handling of anti-recoil sensitivity and setting the position/dimensions of OCR scan block.
* Minor internal changes
* README info and progress updates

cc @mgsweet

**Review by mgsweet — APPROVED** [review]

**Pull request merged**